akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/EventLog.scala

```diff
@@ -18,9 +18,16 @@ import akka.persistence.JournalProtocol.ReplayMessages
 import akka.persistence.SnapshotProtocol.LoadSnapshot
 import akka.persistence._
 
-/** INTERNAL API */
+/**
+ * Actions in this trait support interactions with the event log itself,
+ * which supports replay and recovery. We replay events, for example,
+ * when algorithms change and data/events need to be re-run against them,
+ * or on failure and recovery.
+ *
+ * INTERNAL API
+ */
 @InternalApi
-private[akka] trait JournalInteractions[C, E, S] {
+private[akka] trait EventLog[C, E, S] {
```

Comment:
I don't think this is an improvement, because it implies that this _is_ the event log, while the old name implied that it was a set of interactions with the journal (which was actually the journal and the snapshot plugin).

I'd vote for keeping `JournalInteractions` and introducing a `SnapshotStoreInteractions`.

Reply:
Works for me because my initial want was to separate them out for the two. But given `JournalInteractions` I'd vote for `SnapshotInteractions`, though I am not a fan of the `Interactions` part, which is not important.

Reply:
Sounds good to me!